Configure

```diff
@@ -590,7 +590,7 @@ while ((my $first, my $second) = (shift @list, shift @list)) {
 
 &usage if ($#ARGV < 0);
 
-# For the "make variables" CINCLUDES and CDEFINES, we support lists with
+# For the "make variables" CPPINCLUDES and CPPDEFINES, we support lists with
```

Comment:
Why do we do this only for `CPPINCLUDES` and not `CINCLUDES`?

Reply:
CINCLUDES is a typo